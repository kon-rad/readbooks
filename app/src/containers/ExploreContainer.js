// @flow

import React from 'react';
import ExploreSearch from '../components/ExploreSearch/ExploreSearch';
import ExploreFeed from '../components/ExploreFeed/ExploreFeed';

type Props = {};

class ExploreContainer extends React.Component<Props> {

  render() {
    return (
      <div>
        <ExploreSearch />
        <ExploreFeed />
      </div>
    )
  }
}

export default ExploreContainer;