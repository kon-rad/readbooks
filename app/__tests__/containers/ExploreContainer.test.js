import React from 'react';
import ExploreContainer from '../../containers/Explore';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

describe('ExploreContainer', () => {
  it('should render without throwing', () => {
    shallow(<ExploreContainer/>);
  });

  it('renders correctly', () => {
    const tree = renderer
      .create(<ExploreContainer/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  })
});