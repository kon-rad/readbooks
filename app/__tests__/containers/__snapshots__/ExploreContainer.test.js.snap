// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExploreContainer renders correctly 1`] = `
<div>
  <div
    className="search_container"
  >
    <h3>
      Find interesting reads
    </h3>
    <div
      className="titleSearchWrapper"
    >
      <label
        className="titleSearchLabel"
        htmlFor="titleSearch"
      >
        Search By Title:
      </label>
      <div
        className="autoCompWrapper"
      >
        <input
          autoComplete="off"
          className="searchInput"
          id="titleSearch"
          onChange={[Function]}
          onKeyDown={[Function]}
          type="search"
          value=""
        />
        <div
          className="autoCompList"
          role="list"
          style={
            Object {
              "display": "none",
            }
          }
        />
      </div>
      <input
        className="titleSearchSubmit"
        type="submit"
        value="submit"
      />
    </div>
  </div>
  <div />
</div>
`;
